cmd/sourced/cmd/root.go

```diff
@@ -35,5 +41,14 @@ type Command struct {
 // Execute adds all child commands to the root command and sets flags appropriately.
 // This is called by main.main(). It only needs to happen once to the rootCmd.
 func Execute() {
-	rootCmd.RunMain()
+	if err := rootCmd.Run(os.Args); err != nil {
+		if workdir.ErrWrongWorkdir.Is(err) {
```

Comment:
nit pic: `workdir.ErrWrongWorkdir` -> `workdir` repeated 2 times in the name of package an in the name of error.